llpc/test/shaderdb/general/PipelineVsFs_TestIndirectResourceLayout.pipe

```diff
@@ -0,0 +1,69 @@
+; Check the resource layout mode. In "Indirect" mode, "PushConst" resource is not in root node.
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
```

Comment:
I'm a bit confused. I don't see `--resource-layout-scheme` specified here, doesn't  this test default to the compact mode then?